libs/components/popovers/testing/src/dropdown/harness/dropdown-menu-harness.ts

```diff
@@ -0,0 +1,116 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyDropdownItemHarness } from './dropdown-item-harness';
+import { SkyDropdownMenuHarnessFilters } from './dropdown-menu-harness.filters';
+
+/**
+ * Harness for interacting with dropdown menu in tests
+ */
+export class SkyDropdownMenuHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = '.sky-dropdown-menu';
+
+  #getDropdownItems = this.locatorForAll(SkyDropdownItemHarness);
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyDropdownMenuHarness` that meets certain criteria
+   */
+  public static with(
+    filters: SkyDropdownMenuHarnessFilters
+  ): HarnessPredicate<SkyDropdownMenuHarness> {
+    return SkyDropdownMenuHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Gets the aria-labelledby value
+   */
+  public async getAriaLabelledBy(): Promise<string | null> {
+    return (await this.host()).getAttribute('aria-labelledby');
+  }
+
+  /**
+   * Clicks the dropdown menu item at index
+   * @param index Index of dropdown menu item
+   */
+  public async clickItemAtIndex(index: number): Promise<void> {
+    await (await this.getItemAtIndex(index))?.click();
+  }
+
+  /**
+   * Clicks the dropdown menu item with a specific role
+   * @param role Role of dropdown menu item to be clicked
+   */
+  public async clickItemWithRole(role: string): Promise<void> {
```

Comment:
Could you provide more context on this method's use case?

Reply:
I wrongly assumed that dropdown menu items that could be clicked to open another menu would have `role` set to `menu` but I rechecked the WCAG and mdn docs and all menu items would have `menuitem` role. and don't think we need this anymore. 

We currenly default to `menuitem` but we do allow consumers to change that aria role if they want to. Should we still give them the ability to do so? The only other acceptable roles for a menu item are for checkboxes and radio button menu items.